webrtc.html

```diff
@@ -3881,33 +3882,18 @@ <h4>Updating the Negotiation-Needed flag</h4>
             <p>Set <var>connection</var>.<a>[[\NegotiationNeeded]]</a> to
             <code>true</code>.</p>
           </li>
-          <li>
-            <p data-tests="">[= Chain =] a step to queue a
-            task that runs the following steps, to <var>connection</var>'s
-            [= operations chain =]:</p>
-            <ol>
-              <li>
-                <p class="needs-tests">If <var>connection</var>.<a>[[\IsClosed]]</a>
-                is <code>true</code>, abort these steps.</p>
-              </li>
-              <li class="needs-tests">
-                <p>If <var>connection</var>'s [= signaling state =] is not
-                {{RTCSignalingState/"stable"}}, abort these steps.</p>
-              </li>
-              <li>
-                <p>If <var>connection</var>.<a>[[\NegotiationNeeded]]</a>
-                is <code>false</code>, abort these steps.</p>
-              </li>
-              <li data-tests="RTCPeerConnection-onnegotiationneeded.html">
-                <p>[= Fire an event =] named {{negotiationneeded}}
-                at <var>connection</var>.</p>
-              </li>
-            </ol>
-            <p class="note">This queueing prevents {{negotiationneeded}} from
-            firing prematurely, in the common situation where multiple
-            modifications to <var>connection</var> are being made at once.</p>
+          <li data-tests="RTCPeerConnection-onnegotiationneeded.html">
+            <p>[= Fire an event =] named {{negotiationneeded}}
+            at <var>connection</var>.</p>
```

Comment:
The check against RTCSignalingState being not-"stable" is now in the chain only. Is that correct?

The check against NegotiationNeeded is now unnecessary, since NegotiationNeeded is set directly above in the same task, but it also means that NegotiationNeeded is potentially false for a "long" period while we're waiting for the chain to clear. Is that OK? (if multiple transitions false -> true are chained, only the first will fire a signal, so I suppose that's OK)

This is complicated......